### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by slimooo** [retypeapp/retype#18] [Q&A] [answered]
It's possible to translate in other languages ?

**Comment by fabriciomurta** [comment]
Hi! We've been discussing this and implementing multiple-language support for the interface is in our to-do list.

**Comment by fabriciomurta** [comment, marked as answer]
We've just created issue #24 to track this feature so make sure to subscribe to it to receive updates once we implement multi-language support.

What language(s) would you be interested in by the way?

**Reply by slimooo** [reply]
French for me, I can translate if you want :)

**Reply by alejandroiam** [reply]
I can help translate into Spanish.

**Reply by jereef** [reply]
> I can help translate into Spanish.

me too!

**Reply by geoffreymcgill** [reply]
Thanks everyone with offers to help translate. We are making progress on the ability to set custom string values for the entire interface, although much more work is required. We want to make it as easy as possible to configure a new language.

I'm not sure if the functionality will be included in the `v1.9.0` release but it is a super important feature for us and our most highly requested feature, so we have been making this a high priority feature. It just takes a lot of planning and testing to implement. Good progress is being made. Hang in there.

**Reply by unicorn64** [reply]
I can help translate into Turkish.

**Reply by jorikfon** [reply]
Hello. I can help to translate into Russian

**Reply by composite** [reply]
I can help with Korean. yep. South Korea.

**Reply by jrubio9** [reply]
Hey there! I'm just starting with retype. Congrats for your work, it's so easy and clear.
And the translation feature would be amazing! I can help with Catalan/Spanish if you need! :)

**Reply by ashkheid-zz** [reply]
and I'd be thrilled to help with the Persian realm; but I should wait for the [RTL-challenge](https://github.com/retypeapp/retype/issues/24#issuecomment-885923442) to be passed in the first place. 😎🤞🏻

**Reply by VTenoris** [reply]
Hello. I can help translate into the Turkish

**Reply by geoffreymcgill** [reply]
Retype v3.0 will include support for changing the interface language. Currently 24 languages are supported. 

[French](https://github.com/retypeapp/retype-translations/blob/main/src/UserInterface.fr.resx), [Spanish](https://github.com/retypeapp/retype-translations/blob/main/src/UserInterface.es.resx), [Korean](https://github.com/retypeapp/retype-translations/blob/main/src/UserInterface.ko.resx), [Russian](https://github.com/retypeapp/retype-translations/blob/main/src/UserInterface.ru.resx), and [Turkish](https://github.com/retypeapp/retype-translations/blob/main/src/UserInterface.tr.resx) are all supported. Farsi is not (sorry). 

See the following for more comments:
https://github.com/retypeapp/retype/issues/404#issuecomment-1469152692

See the following docs for the translations:
https://retypeapp.github.io/retype-translations/

The source repo is available at:
https://github.com/retypeapp/retype-translations

RTL is not support but hopefully one day.

**Reply by geoffreymcgill** [reply]
The new `locale` project configuration is now available in Retype v3.0, see [docs](https://retype.com/configuration/project/#locale).

The Retype Translation repo is available at https://retypeapp.github.io/retype-translations/. 

Hope this helps.
